salt/vault/map.jinja

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls syntax=yaml softtabstop=2 tabstop=2 shiftwidth=2 expandtab autoindent
+
+{% import_yaml "vault/maps/defaults.yaml" or {} as defaults %}
+{% import_yaml "vault/maps/osfamilymap.yaml" or {} as osfamilymap %}
+{% import_yaml "vault/maps/initfamilymap.yaml" or {} as initfamilymap %}
+
+{%- set vault = salt.grains.filter_by(defaults,
+    default='vault',
+    merge=salt.grains.filter_by(osfamilymap, grain='os_family',
+      merge=salt.grains.filter_by(initfamilymap, grain='init',
+        merge=salt.pillar.get('vault:lookup', default={})
+      )
+    )
+) %}
+
+{#- Merge the vault pillar #}
+{%- set vault = salt.pillar.get('vault', default=vault, merge=True) %}
```

Comment:
I don't think this line is doing anything we need... The pillar merge we need is already happening in the prior line...